Create ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md (#2533)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md

---
date: ""

> title: Logging out all users when deleting/changing global roles.

Reviewer: ```suggestion
title: Logging out all users when deleting or modifying a global role
```

> title: Logging out all users when deleting or modifying a global role

Reviewer: ```suggestion
title: Option to log out all users when deleting or modifying a global role
```

> title: cherry-pick of all feature: 31cccab4f43f2bcc51a132dcb402b1a… (#7309)

Author: The Claude AI suggested to use the following title:
```suggestion
title: All feature cherry-picked from main branch
```

product_area: Platform services
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-56510
version: 1020.36.0
---

> cherry-pick of all feature: 31cccab4f43f2bcc51a132dcb402b1a… (#7309)

Author: The Claude AI suggested to use the following body:
```suggestion
As part of ongoing development efforts, a commit containing an "all feature" was made on the main branch. To ensure this feature is also available on the current branch, the commit has been cherry-picked and merged. This change brings the current branch up-to-date with the latest "all feature" from the main branch. End-users should not experience any noticeable impact, as this is an internal development change.
```

Reviewer 2: @mmed-c8y please verify the suggestions as current release note is to technical

> When deleting or modifying a global role, the administrator has the option to log out all users assigned to that role.

Reviewer 2: ```suggestion
When deleting or modifying a global role, the administrator has the option to log out all users who are assigned to that role.
```

Reviewer: I suggest to add "now": the administrator now has the option...

> When deleting or modifying a global role, the administrator has the option to log out all users who are assigned to that role.

Reviewer: ```suggestion
When deleting or modifying a global role, the administrator has the option to log out all users that are assigned to that role.
```

> When deleting or modifying a global role, the administrator has the option to log out all users that are assigned to that role.

Reviewer: ```suggestion
When deleting or modifying a global role, the administrator now has the option to log out all users that are assigned to that role.
```

> When deleting or modifying a global role, the administrator now has the option to log out all users that are assigned to that role.

Reviewer: @saran-govindarajan @JanePorter This information is quite poor. What was the issue the user was facing and what is the advantage of this option? We need more details here to make this entry of any use to the user. This PR is now open for a couple of weeks, actually change log PR should be merged asap.

Reviewer 2: Actually, it seems that the description is complete - previously there was no option to log all users out when modifying a global role, but now there is. Optionally, we can add information about automatic refresh of roles when a relogin is forced.

Reviewer 3: @kowa-c8y but why would someone want to lot out all users? Should it read something like:

When deleting or modifying a global role an administrator can now ensure the changes take immediate effect by using a new option which will log out all users assigned to that role.

Reviewer 2: Good point! The intention here is definitely worth mentioning.  :+1:

Reviewer 2: @BeateRixen  please check last commit and if its ok please merge it

> When deleting or modifying a global role, the administrator now has the option to log out all users that are assigned to that role.

Reviewer 2: ```suggestion
When deleting or modifying a global role, an administrator can now ensure that changes take immediate effect by using a new option to log out all users assigned to that role.
```

Reviewer 2: @BeateRixen please check if it's ok commit and we will merge it :+1:
